static-environments/coverage-e2e-env/.gitignore

```diff
@@ -0,0 +1,3 @@
+coverage
+node_modules
+.code-pushup
```

Comment:
Not sure this is needed, our root `.gitginore` already includes these patterns.

Reply:
refactored to dynamic setup